src/Model/AuthorManager.php

```diff
@@ -23,4 +25,12 @@ public function insertAuthor(array $comicAuthor): void
             $statement->bindValue(':biography', $comicAuthor['biography'], \PDO::PARAM_STR);
             $statement->execute();
     }
+
+    public function deleteAuthor(int $authorId)
+    {
+        $query = 'DELETE FROM ' . self::TABLE . ' WHERE id=:id;';
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue(':id', $authorId, \PDO::PARAM_INT);
+        $statement->execute();
+    }
```

Comment:
No. There already is a delete() function in AbstractManager who do exactly the same thing. We don't duplicate code.